build/libevhtp/evhtp.patch

```diff
@@ -0,0 +1,42 @@
+diff --git a/evhtp.c b/evhtp.c
```

Comment:
Did you add the patch file intentionally?

Reply:
Yes. We must patch evhtp so that it collects the HTTP body into a contiguous memory buffer.